src/app/templates/RevealSecrets/SecretField.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC, useMemo } from 'react';
+
+import { Alert } from 'app/atoms/Alert';
+import { ReadOnlySecretField } from 'app/atoms/ReadOnlySecretField';
+import { TID, T } from 'lib/i18n';
+
+import { RevealSecretsSelectors } from './RevealSecrets.selectors';
+
+interface Props {
+  revealType: 'private-key' | 'seed-phrase';
+  value: string;
+}
+
+interface Texts {
+  title: TID;
+  description: React.ReactNode;
+  attention: TID;
+}
+
+export const SecretField: FC<Props> = ({ revealType, value }) => {
+  const texts = useMemo<Texts>(() => {
+    switch (revealType) {
+      case 'private-key':
+        return {
+          title: 'privateKey',
+          description: <T id="privateKeyFieldDescription" />,
+          attention: 'doNotSharePrivateKey'
+        };
+
+      case 'seed-phrase':
+        return {
+          title: 'seedPhrase',
+          description: (
+            <>
+              <T id="youWillNeedThisSeedPhrase" /> <T id="keepSeedPhraseSecret" />
+            </>
```

Comment:
```suggestion
           
              <T id="youWillNeedThisSeedPhrase" /> <T id="keepSeedPhraseSecret" />
           
```

Reply:
@alex-tsx let's add a line break between those two, so it's easier to distinguish separate components (as it indeed may seem like one from the first glimpse)